src/kernels/execution/cellExecutionMessageHandler.ts

```diff
@@ -903,9 +903,11 @@ export class CellExecutionMessageHandler implements IDisposable {
 
     private handleError(msg: KernelMessage.IErrorMsg) {
         let traceback = msg.content.traceback;
+        traceInfoIfCI(`Traceback for error ${traceback}`);
         this.formatters.forEach((formatter) => {
             traceback = formatter.format(this.cell, traceback);
         });
+        traceInfoIfCI(`Traceback for error after formatting ${traceback}`);
```

Comment:
Found that on CI sometimes things fail even before this PR, hence added for CI only